src/components/frame/LandingPage/ColonyCards/CreateNewColonyCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { Plus } from '@phosphor-icons/react';
+import React, { useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Button from '~v5/shared/Button/Button.tsx';
+
+import BaseColonyCard from './BaseColonyCard/BaseColonyCard.tsx';
+
+const displayName = 'frame.LandingPage.ColonyCards';
+
+export interface ColonyCreateCardProps {
+  invitationsRemaining: number;
+  onCreate: () => void;
+}
+
+const MSG = defineMessages({
+  remaining: {
+    id: `${displayName}.remaining`,
+    defaultMessage: `{remaining} {remaining, plural, one {INVITATION} other {INVITATIONS}} REMANING`,
```

Comment:
The wording here should be "colonies" remaining, not "invitations".

<img width="178" alt="Screenshot 2024-10-28 at 10 38 15" src="https://github.com/user-attachments/assets/4b7bec8c-c883-4b56-b50d-ca65a0d42950">